lib/direct-message-key.js

```diff
@@ -0,0 +1,19 @@
+// WIP
```

Comment:
A shared DM Key is needed for e.g. Mix and Christian to do direct messaging.
For a first pass we decided to do what we've done in the past - just sclarar-mult e.g. Mix Private key * Christian Public key.

This gives us a shared key which we can then use to pass into private group encryption.

---

FYI, private-groups uses the same `recps` field , but it reads the recps and maps those recps into keys before doing encryption. e.g. 
- `@<feedId>=.ed25519` >> this shared DM key
- %<groupId>=.cloaked` >> the shared symmetric key for the group